content/en/docs/reference/access-authn-authz/rbac.md

```diff
@@ -285,6 +285,29 @@ If you restrict `list` or `watch` by resourceName, clients must include a `metad
 For example, `kubectl get configmaps --field-selector=metadata.name=my-configmap`
 {{< /note >}}
 
+Rather than referring to individual `resources` and `verbs` you can use the wildcard `*` symbol to refer to all such objects.
+For `nonResourceURLs` you can use the wildcard `*` symbol as a suffix glob match and for `apiGroups` and `resourceNames` an empty set means that everything is allowed.
+Here is an example that 
+
+```yaml
+apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
I would add a big “do not use this role it's an example” comment. As well as the caution callout.

(also, if people did want a similar role, they probably have `cluster-admin` built in).

Reply:
Ah yes good point - updated in https://github.com/kubernetes/website/pull/36043/files#diff-b39dd9c0a047fe6b4d12eb238bc7d66137d6828df7c3a03801c5d3e375ad8a42

Reply:
looks good to me